pkg/collect/certificates.go

```diff
@@ -114,36 +114,35 @@ func configMapCertCollector(configMapName string, namespace string, client kuber
 			SecretName: configMapName,
 			Namespace:  namespace,
 		}
-		trackErrors = append(trackErrors, "Either the configMap does not exist in this namespace or RBAC permissions are preventing certificate collection")
+		trackErrors := append(trackErrors, "Either the configMap does not exist in this namespace or RBAC permissions are preventing certificate collection")
```

Comment:
Its better to write the actual error in the `trackErrors` list i.e `err.Error()`. The error objects [have decent messages](https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L70)

Reply:
Changed to err.Error()